src/client/containers/Profile/index.js

```diff
@@ -1,5 +1,65 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUserFirebaseToken } from '../../firebase/auth';
+import man from '../../assets/images/portraits/man1.jpg';
+import './index.styles.css';
+import { ProfileCardComponent } from '../../components/ProfileCardComponent/ProfileCardComponent';
+import { YourUploadedCVs } from '../../components/MyProfile/YourUploadedCVs/YourUploadedCVs';
+import TitleCvCard from '../../components/MyProfile/TitleCvCard/TitleCvCard';
 
 export default function Profile() {
-  return <div>This is your private profile</div>;
+  const [cvsList, setCvsList] = useState([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  // Here it needs the endpoint to get users' infor like github or  profile image url
+  useEffect(() => {
```

Comment:
@pdsorensen I tried to reach to the firebase token here, because we have item like this  ```"firebase_token": "abc12d"``` in the ```/api/users```then I can pick them up for the ```<ProfileCardComponent>``` such as ```profileImageUrl```  ,```linkedin``` , ```fullName``` and so on.